tests/integration-tests/configs/develop.yaml

```diff
@@ -66,6 +66,17 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["alinux2"]
           schedulers: ["slurm"]
+  create:
+    test_create.py::test_create_wrong_os:
+      dimensions:
+        - regions: ["eu-central-1"]
+          instances: {{ common.INSTANCES_DEFAULT_X86 }}
```

Comment:
you are specifying an instance here but the test function is not using the fixture. Same below

Reply:
I am setting instance in the config files using this dimension. Should I use the fixture elsewhere in the test?

Reply:
I think for some reason I just missed the fact that you were passing instance as an arg to the test function.